src/test/java/org/folio/rest/impl/PatronResourceImplTest.java

```diff
@@ -1020,6 +1031,61 @@ public final void testPostPatronAccountByIdInstanceByInstanceIdHoldWithErrors(
     logger.info("Test done");
   }
 
+  @Test
+  public final void testPostPatronAccountByIdInstanceByInstanceIdHold422Error() {
```

Comment:
I think the name of the test should be a summary of the expected behaviour. What do you think of the suggestion below?

```suggestion
  public final void postToCreateInstanceHoldRequestShouldForwardOn422Error() {
```

Reply:
Thank you, will fix it.